contribution/k3s/install_k3s.sh

```diff
@@ -3,7 +3,7 @@
 # Copyright 2021 Authors of KubeArmor
 
 # create a single-node K3s cluster
-if [ -x "$(command -v docker)" ]; then # docker
+if [ "$RUNTIME" == "docker" ]; then # docker
```

Comment:
who defines $RUNTIME? is it possible to detect cri-o automatically?

Reply:
@nam-jaehyun right now RUNTIME would be defined while executing the script.
To detect automatically, we may check for existence of socket files or run commands to check for a runtime as we were doing for docker.
However I thought it might be not so flexible. Many systems like GitHub actions come with docker pre-installed and running so in that case this script would always use docker.